README.md

```diff
@@ -18,32 +18,32 @@ cargo install --locked --git https://github.com/r0gue-io/pop-cli
 
 ### Parachains
 
-Use `pop` to create a new Parachain project:
+Use `pop` to create a new Parachain project.
+To be guided through the entire parachain creation process, simply execute
+```sh
+pop new parachain
+```
 
+If no guidance is needed, proceed with:
 ```sh
 # Create a minimal parachain
 pop new parachain my-app
-# Get a pallet-contracts enabled parachain
-pop new parachain my-app cpt
-# Get a evm compatible parachain
-pop new parachain my-app fpt
 ```
 
-Use `pop` to build your Parachain:
 
+We also integrate other provider templates in the tool, check them running:
 ```sh
-# Build your parachain
-pop build parachain -p ./my-app
+pop new parachain --help
 ```
-
-or
-
+Some examples are: 
 ```sh
-cd my-app
-pop build parachain
+# Get Parity's pallet-contracts enabled parachain template
+pop new parachain my-app parity -t cpt
+# Get Parity's evm compatible parachain template
+pop new parachain my-app parity -t fpt
```

Comment:
For me no, is more clear to pick the provider and then the template associate to it,  but maybe would be good to open a discussion and see users feedback.

Ideally would be to don't even have to add the flag -t:
`pop new p my-app parity cpt`

Reply:
For a new user, they might not feel this as intuitive as it could've been. If I am new, I would want to start a new parachain without knowing the additional concept of a provider (which in this case seems mandatory). In such a case a default should've been selected for me.

Reply:
There is a default provider already, `pop`.
If you are new and don't want to use a specific provider, you don't have to know about that, no problem.
`pop new parachain my-app ` will generate the base parachain and `pop new parachain my-app -t contracts` the contracts parachain from Pop (When it exists)